packaging/systemd/microshift.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=MicroShift
-Wants=network-online.target crio.service openvswitch.service microshift-ovs-init.service
-After=network-online.target crio.service openvswitch.service microshift-ovs-init.service
+Wants=network-online.target crio.service
+After=network-online.target crio.service
```

Comment:
I think we should allow user to use OVN-k also (upstream component , build for OKD) and different CNI plugin (flannel in this case) and this different CNI plugin should have updated unit file instead we doing it here.

Reply:
yea, its not going to be merged like that,  yes that part will be handled by the CNI (flannel).
I'll remove it .